src/material-experimental/mdc-button-toggle/button-toggle.scss

```diff
@@ -0,0 +1,5 @@
+@use '@material/ripple/common';
+@use '@material/segmented-button/styles';
```

Comment:
I believe it has both because the theme is included here https://github.com/material-components/material-components-web/blob/master/packages/mdc-segmented-button/segment/_segment.scss#L33

Reply:
We don't want both here, this should be only the structural styles (see other MDC-based components for reference)